### DISCUSSION TITLE
UI 10 > text field  - value insteead of ID

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17100] [How to] [answered]
Got workflow which suppose to copy name of user to text field. Currently WF returning user ID. How to get the name of user?

**Comment by rskrzypczak** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/15288

**Reply by s2b-git** [reply]
forgot it was touched.
